pages/shopping-cart/index.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import ShoppingCart from "../../components/ShoppingCart";
+import Layout from "/components/Layout";
+import { useContext } from "react";
+import { CartContext } from "../_app";
+
+
+
+
+
+export default function ShoppingCartPage() {
+
+    
+  return (
+    <>
+    
+      <Layout />
+      <ShoppingCart />
+     
+    </>
```

Comment:
Just delete empty space.

```suggestion
    <>
      <Layout />
      <ShoppingCart />
    </>
```